### ISSUE TITLE
add host to config/foam.conf.php?

### ISSUE THREAD

**Issue by lachlan-00** [Andrew-McGee/foam#13]
I've moved the hosting for demo (https://demo.ampache.dev/foam/) and develop (https://develop.ampache.dev/foam/) and i've put in the default demo details in auth.php
![image](https://user-images.githubusercontent.com/1305249/198506276-cf4f4fca-760a-4ec9-ad15-30473497bfae.png)

I know it's simple to do to auth. but could that be added into the conf file?

**Comment by Andrew-McGee** [comment]
Hey mate - good to hear from you. Apologies I haven't been super responsive - work is crazy. But I am still chipping away here.

Yes good idea I can default credentials into the conf file. I have a todo item to fix this whole login screen so can include that too.

**Comment by Andrew-McGee** [comment]
I noticed something weird on your demo. The Supamix button doesn;t work. If you use my demo page and connect to your same ampache host it works fine: http://foamdemo.mcgee.technology/

Maybe there is a difference in PHP settings. It could time out - supamix takes a few seconds to randomly build 100 songs from multiple albums.

**Comment by Andrew-McGee** [comment]
Added to the conf file - you can populate or leave blank and it caters for both scenarios.

`  // Default login information that will be populated on the login form`
`  // If left blank then fields will be unpopulated except for placeholder hints`
`  $defHost = '';  // Eg. http://myhost.com or myampachehost.com or IP equivalent. Can include port number.`
`  $defUser = '';  // Eg. The Ampache username to default`
`  $defPass = '';  // Eg. The Ampache password to default use (WARN: this leaves it wide open for anyone to login)`

**Comment by lachlan-00** [comment]
i get a php error for supamix
```
[Mon Oct 31 20:26:51.762894 2022] [proxy_fcgi:error] [pid 1269354:tid 140678996842240] [client 58.84.188.56:40344] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught Error: Undefined constant "song" in /var/www/demo/demo.ampache.dev/public/foam/supamix_tracks.php:18\nStack trace:\n#0 {main}\n  thrown in /var/www/demo/demo.ampache.dev/public/foam/supamix_tracks.php on line 18', referer: https://demo.ampache.dev/foam/albums_view.php
```

Haven't looked at the php file yet but i'll see what i can see in the morning.

**Comment by lachlan-00** [comment]
I've been migrating the api5 out to a separate subclass for api6 so the error is likely me. had to make a few changes to fix things

**Comment by Andrew-McGee** [comment]
Yep the undefined constant is an array that doesn't have quotes. I tried fixing some other bits with it and completely borked it tonight so I will have to wait until later in the week to fix.

**Comment by Andrew-McGee** [comment]
Supamix working. All arrays updated for PHP 8+ syntax. Temp fix using ampache API version=5.6.0 for auth issues.
